### DISCUSSION TITLE
Contacts not showing

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#884] [General]
Hi everyone, here is a video which will illustrate what seems to be a bug with contacts not showing. 
https://www.loom.com/share/bac5bf14c9644d04b1f06b8083be2b78

**Comment by kiyote33** [comment]
Hi Julien, I'm looking into this, but looking at the code it appears by-design that the contacts only display for logged-in users. I will attempt to confirm this with the rest of the team, and get back to you soon.
